core/src/main/java/com/yahoo/oak/BasicChunk.java

```diff
@@ -241,6 +246,11 @@ boolean overwriteExistingValueForMove(ThreadContext ctx, V newVal) {
     static class Statistics {
         private final AtomicInteger addedCount = new AtomicInteger(0);
         private int initialCount = 0;
+        private static final int VALS_4_AVERAGE_SIZE = 2048;
```

Comment:
I think it was taken from one of the previous versions of the code. We decided to delete this usage. Please remove unless needed for your code correctness